data.tf

```diff
@@ -8,15 +8,14 @@ data "aws_ec2_instance_type" "this" {
 
 data "aws_ami" "ami" {
   most_recent = true
-  owners      = ["099720109477"]
-
   filter {
     name   = "name"
     values = [var.ami_name]
   }
+}
 
-  filter {
-    name   = "virtualization-type"
-    values = ["hvm"]
```

Comment:
I hadn't seen this way of validating before, very useful since the older TF versions dont allow complex input validation. Thanks!